javascript_nodejs/14.nlp-with-dispatch/dialogs/homeAutomation/home-automation-state.js

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const HOME_ATUOMATION_STATE_PROPERTY = 'homeAutomation.state';
+
+class DeviceState {
+    /**
+     * 
+     * @param {String} device 
```

Comment:
As code / parameter comments go, there is room here for significant improvement.

Reply:
Improved. For simplicity, I'd like to keep these all strings. This way the sample is cleaner.